adding_service_cluster/adding-service.adoc

```diff
@@ -6,9 +6,9 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
```

Comment:
I would defer to someone on the service delivery team but the logging addon has been deprecated and no longer offered.  I did see in telemetry there are some deployments but very few

Reply:
Let's maybe open a Jira to follow up on this separately. I want to make sure we don't widen the scope of this PR too much, and this file was only really modified to make sure the link stayed up to date when the other logging content was moved.